"use server";

export const signIn = async () => {
  try {
  } catch (error) {
    console.error("Error in signIn function", error);
  }
};
export const signUp = async ({ userdata }: SignUpParams) => {
  try {
  } catch (error) {
    console.error("Error in signUp function", error);
  }
};
